Add an explicit coercion so that it compiles with both lablgtk 2.16 and 2.18.

diff --git a/src/ide/gmain.ml b/src/ide/gmain.ml
index 46756f4..fb5980f 100644
--- ./src/ide/gmain.ml
+++ ./src/ide/gmain.ml
@@ -801,7 +801,7 @@ let info_window ?(callback=(fun () -> ())) mt s =
   in
   let d = GWindow.message_dialog
     ~message:s
-    ~message_type:mt
+    ~message_type:(mt :> Gtk.Tags.message_type)
     ~buttons
     ~title:"Why3IDE"
     ~icon:(!Gconfig.why_icon)
